torch/jit/_pickle.py

```diff
@@ -0,0 +1,36 @@
+import torch
+import functools
+import pickle
+
+
+class TensorID(object):
+    def __setstate__(self, id):
+        self.id = id
+
+
+class IntList(object):
+    def __setstate__(self, data):
+        self.data = data
+
+
+class LiteralTensor(object):
```

Comment:
This seems unused in this PR. Hard to know if it would work or not. Best to add it later.